vue/sbc-common-components/src/util/config-helper.ts

```diff
@@ -0,0 +1,17 @@
+export default class ConfigHelper {
```

Comment:
If you changed this to a mixin then you wouldn't have to call each method as "ConfigHelper.xxx()" 😁 

But I DON'T know how to use a mixin in a non-Vue class 😒

Reply:
Might be nice, but not doing this for now.

Reply:
np